### ISSUE TITLE
404 on bundle.js on triage pages

### ISSUE THREAD

**Issue by meandavejustice** [nodebugme/site#1]
`GET https://media.nodebug.me/public/bundle.js`

Happens while triaging any bugs https://nodebug.me/triage/joyent/node/*

**Comment by meandavejustice** [comment]
Doesn't seems to break any functionality, may have just been a forgotten script tag?

**Comment by chrisdickinson** [comment]
Yeah -- it's a placeholder for browserified assets, but it turns out there's no frontend JS on the site to speak of. It should be removed!
